sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventGroupController.java

```diff
@@ -203,19 +202,21 @@ public CommitDiscardWrapperComponent<?> getEventGroupEditComponent(String uuid)
 			});
 		}
 
-		if (user.hasUserRight(UserRight.EVENTGROUP_DELETE) && hasRegion) {
-			editView.addDeleteListener(() -> {
-				deleteEventGroup(eventGroup);
-				UI.getCurrent().getNavigator().navigateTo(EventsView.VIEW_NAME);
-			}, I18nProperties.getString(Strings.entityEventGroup));
-		}
+		// TODO Temporarily deleted in #8915, enable again in #8851
```

Comment:
Write this marker like this:
"TODO #8851: Enable temporarily deleted button when EventGroup deletion works"